misc/acrn-config/host/kconfig/test.py

```diff
@@ -0,0 +1 @@
+print('aaaaaaaaaaaa')
```

Comment:
why aaaa?

Reply:
Delete this print line, which was for temporary debug. 
PS: Actually kconfig/ folder was just copied from original acrn-hypervisor/scripts. pylint is only 7.76 score, i will try best to clean most basic coding guideline issues in this PR, and continue optimizing target to 10 score in another patch/PR.

Reply:
Yes, let us close this PR first. Revise to fix the issue and re-submit